assets/component-facets.css

```diff
@@ -518,6 +518,21 @@ a.active-facets__button.focused .active-facets__button-inner,
   max-width: 37.5rem;
   display: flex;
   flex-direction: column;
+  --shadow-horizontal-offset: var(--popup-drawer-shadow-horizontal-offset);
+  --shadow-vertical-offset: var(--popup-drawer-shadow-vertical-offset);
+  --shadow-blur-radius: var(--popup-drawer-shadow-blur-radius);
+  --shadow-opacity: var(--popup-drawer-shadow-opacity);
+  border-bottom-left-radius: var(--popup-drawer-corner-radius);
+  border-top-left-radius: var(--popup-drawer-corner-radius);
+  border-color: rgba(var(--color-foreground), var(--popup-drawer-border-opacity));
+  border-style: solid;
+  border-width: 0 0 var(--popup-drawer-border-width);
```

Comment:
Should there be a border around the filter drawer like there is on the menu nav ?
Currently it looks like this: ![](https://screenshot.click/21-11-26l1n-z5vpx.png)

Reply:
Let's apply the border to the opposite side of the device for full height element. We'll be sketching more of these soon to capture the right outcome. 🙏